experimental/FTheoryTools/src/LiteratureModels/Models/model1408_4808-11-WSF.json

```diff
@@ -46,14 +46,14 @@
     },
     "model_data": {
         "base_dim": 3,
-        "model_sections": ["s5", "s9"],
-        "base_coordinates": ["s5", "s9", "s1", "s2", "s3", "s6"],
+        "divisors_user_must_specify": ["s5", "s9"],
+        "divisors_parametrized_by_user_specified_divisors": ["s1", "s2", "s3", "s6"],
```

Comment:
This is a point we have discussed some elsewhere, but I'll raise it again here for visibility. In a future PR, this file and other will be updated so that the user-given divisor classes are always S7 and S9. This model does not have an s7 parameter, but does have an s9 parameter, but since `divisors_parametrized_by_user_specified_divisors` only contains parameters that don't appear in `divisors_user_must_specify`, the constructor has no way of telling that it *should* choose a section for s9 but *should not* choose a section for s7. I think this is not necessarily harmful, as the chosen section for s7 will just never be used, but it is slightly wasteful.

Listing all divisor classes in this second field would provide the necessary information to only create the desired section. See earlier related comment on "model1109_3454.json"